ci/dash/build_src.sh

```diff
@@ -26,7 +26,9 @@ if [ "$CHECK_DOC" = 1 ]; then
     #test/lint/check-doc.py
     # Run all linters
     test/lint/lint-all.sh
-    test/lint/extended-lint-all.sh
+    # Disable extended linters for now; the only "extended" linter we have is for cppcheck, and we already do all of
+    # that in lint-cppcheck-dash
+#    test/lint/extended-lint-all.sh
```

Comment:
It indeed runs:
```
$ test/lint/lint-all.sh 
test/lint/lint-cppcheck-dash.sh: line 10: a: command not found
```
Let's drop this one completely then

```suggestion
```

Reply:
`test/lint/extended-lint-all.sh` is dropped here: a8098f2ce | Merge bitcoin/bitcoin#25091: test: Remove extended lint (cppcheck)

Which one we are going to backport patially: remove test/lint/extended-lint.all.sh but keep cppcheck

Reply:
resolved in https://github.com/dashpay/dash/pull/6171/commits/81d64e1d5e694b220b3f3edfeda85d7422288e60